debian/patches/0018-focus-object-can-be-null.patch

```diff
@@ -0,0 +1,29 @@
+Index: maliit-framework-0.99.1+git20151118+62bd56b/input-context/minputcontext.cpp
+===================================================================
+--- maliit-framework-0.99.1+git20151118+62bd56b.orig/input-context/minputcontext.cpp	2020-03-26 18:35:00.000000000 -0500
++++ maliit-framework-0.99.1+git20151118+62bd56b/input-context/minputcontext.cpp	2020-03-26 19:00:22.927732927 -0500
+@@ -278,12 +278,14 @@
+ 
+ void MInputContext::setFocusObject(QObject *focused)
+ {
++    if (!focused) return;
+     if (composeInputContext) composeInputContext->setFocusObject(focused);
+     if (debug) qDebug() << InputContextName << "in" << __PRETTY_FUNCTION__ << focused;
+ 
+     updateInputMethodExtensions();
+ 
+     QWindow *newFocusWindow = qGuiApp->focusWindow();
++    if (!newFocusWindow) return;
```

Comment:
This case is already handled in the code below.

Reply:
I don't believe it is. Without this line, the crash in unity8 occurs reproducibly. Every time I plug in a new display, crash.

Reply:
Gotcha!

```patch
     if (!active && currentFocusAcceptsInput) {
         imServer->activateContext();
         active = true;
     }
 
-    if (currentFocusAcceptsInput) {
+    if (newFocusWindow && currentFocusAcceptsInput) {
         updateServerOrientation(newFocusWindow->contentOrientation());
     }
```

Reply:
Swapped that in